cmov/src/x86.rs

```diff
@@ -49,6 +49,7 @@ impl Cmov for u32 {
     }
 }
 
+#[cfg(not(target_arch = "x86"))]
```

Comment:
I think this would be more clearly expressed as:

```suggestion
#[cfg(target_arch = "x86_64")]
```

Reply:
I agree, it's easier to understand for sure (especially because there's only two possible options within this module).